src/post/entities/hashtag.entity.ts

```diff
@@ -0,0 +1,27 @@
+import {
+    Column,
+    DeleteDateColumn,
+    Entity,
+    OneToMany,
+    PrimaryGeneratedColumn,
+    UpdateDateColumn,
+} from 'typeorm';
+import { Hashtag_Post } from './hashtag_post.entity';
+
+@Entity('hashtag')
+export class Hashtag {
+    @PrimaryGeneratedColumn('increment')
+    id: number;
+
+    @Column()
+    tag: string;
+
+    @UpdateDateColumn()
+    updatedAt: Date;
+
+    @DeleteDateColumn()
+    deletedAt: Date | null;
+
+    @OneToMany(() => Hashtag_Post, hashtag_post => hashtag_post.hashtag)
```

Comment:
```suggestion
    @OneToMany(() => Hashtag_Post, (hashtag_post) => hashtag_post.hashtag)
```
공식문서 처럼 () 써주니 빨간줄 안뜨는거 같아요

Reply:
앗 혹시 어떤 에러 뜨시나요??

Reply:
lint 에러 납니다 ㅜㅜ